File: check_msb.py
import requests
import json

try:
    response = requests.get('https://bible.helloao.org/api/available_translations.json')
    data = response.json()
    
    found = False
    for t in data['translations']:
        if t['id'] == 'MSB' or 'Modern' in t['name'] or 'Modern' in t['englishName']:
            print(f"Found: {t['id']} - {t['name']} ({t['englishName']})")
            found = True
            
    if not found:
        print("MSB or Modern Spelling Bible not found in the list.")
        
except Exception as e:
    print(f"Error: {e}")

File: check_msb.js
const https = require('https');

https.get('https://bible.helloao.org/api/available_translations.json', (resp) => {
    let data = '';

    // A chunk of data has been received.
    resp.on('data', (chunk) => {
        data += chunk;
    });

    // The whole response has been received.
    resp.on('end', () => {
        try {
            const json = JSON.parse(data);
            let found = false;
            json.translations.forEach(t => {
                if (t.id === 'MSB' || t.name.includes('Modern') || t.englishName.includes('Modern')) {
                    console.log(`Found: ${t.id} - ${t.name} (${t.englishName})`);
                    found = true;
                }
            });

            if (!found) {
                console.log("MSB or Modern Spelling Bible not found in the list.");
            }
        } catch (e) {
            console.error(e.message);
        }
    });

}).on("error", (err) => {
    console.log("Error: " + err.message);
});
